modal/gpu.py

```diff
@@ -33,7 +31,7 @@ def __init__(
         self,
         count: int = 1,  # Number of GPUs per container. Defaults to 1.
     ):
-        super().__init__(api_pb2.GPU_TYPE_T4, count, "T4")
+        super().__init__(count, "T4")
```

Comment:
Minorly annoying that the order of arguments is reversed from the string spelling (i.e. `"T4:{count}"`).

Reply:
yeah i can flip it